client/src/components/J40MainGridContainer.tsx

```diff
@@ -0,0 +1,30 @@
+// Trussworks GridContainer won't allow it to span 100% of the page, so
+// this works around it and tries to hide the complexity in component
+import React, {ReactNode} from 'react';
+import {GridContainer} from '@trussworks/react-uswds';
+
+interface ILayoutProps {
+  children: ReactNode,
+  fullWidth?: boolean,
+  className?: string
+}
+
+const J40MainGridContainer = ({
+  children,
+  fullWidth = false,
+  className = 'j40-grid-container'}: ILayoutProps) => {
+  return !fullWidth ? (
```

Comment:
Had to think a bit about the `!` - Maybe flipping this would make it a bit easier to follow? 

e.g. 
```js
fullWidth ? <div>... : <GridContainer>
```

Reply:
Fair. I debated flipping it. Done.